pipelines/broad/arrays/multi_sample/MultiSampleArrays.changelog.md

```diff
@@ -1,3 +1,9 @@
+# 1.5.1
```

Comment:
Is this a patch version if outputs are affected?

Reply:
Yeah, I thought about this.  It's very minor changes.  But changes nonetheless, so I'll bump the minor version.